nixos/modules/services/misc/gitlab.nix

```diff
@@ -193,7 +193,7 @@ let
       makeWrapper ${cfg.packages.gitlab.rubyEnv}/bin/rails $out/bin/gitlab-rails \
           ${concatStrings (mapAttrsToList (name: value: "--set ${name} '${value}' ") gitlabEnv)} \
           --set PATH '${lib.makeBinPath [ pkgs.nodejs pkgs.gzip pkgs.git pkgs.gnutar postgresqlPackage pkgs.coreutils pkgs.procps ]}:$PATH' \
```

Comment:
Changing PATH like this is not going to work with makeBinaryWrapper, since `$PATH` won't be expanded at runtime in the binary wrappers (like it is in the bash wrappers). Instead, use --prefix
```sh
--prefix PATH ':' '${lib.makeBinPath [ pkgs.nodejs pkgs.gzip pkgs.git pkgs.gnutar postgresqlPackage pkgs.coreutils pkgs.procps ]}'
```

Reply:
Same thing, this isn't affected.